repl/src/dotty/tools/repl/JLineTerminal.scala

```diff
@@ -33,12 +33,12 @@ class JLineTerminal extends java.io.Closeable {
     builder.build()
   private val history = new DefaultHistory
 
-  private def blue(str: String)(using Context) =
-    if (ctx.settings.color.value != "never") Console.BLUE + str + Console.RESET
+  private def purple(str: String)(using Context) =
+    if (ctx.settings.color.value != "never") Console.MAGENTA + str + Console.RESET
     else str
-  protected def promptStr = "scala"
-  private def prompt(using Context)        = blue(s"\n$promptStr> ")
-  private def newLinePrompt(using Context) = blue("     | ")
+  protected def promptStr = "@"
+  private def prompt(using Context)        = purple(s"\n$promptStr ")
+  private def newLinePrompt(using Context) = purple("  ")
```

Comment:
This is my personal preference. If we were to have a `>` on the first line then maybe it's best to have one also in the lines after that to be able to easily keeps things aligned.
```suggestion
  private def newLinePrompt(using Context) = purple(">")
```

Reply:
The problem with adding `>`s in subsequent lines is it makes copy paste harder. Without the `>` you just copy-paste-dedent, with the `>` you paste and the need to remove the `>` from each subsequent line, and also manually remove the space since it'll be one remaining space so a 2-space dedent doesnt work